.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug report
+about: Create a report to help us improve and fix the bug
+title: "[BUG]"
+labels: bug
+assignees: ''
+
+---
+
+**Describe the bug**
+
+
+**how to Reproduce**
+
+
+**Screenshots**
+
+
+**project link**
+
+
+**error code**
```

Comment:
I would remove the error code section because the website will auto generate this if it has an error. Manual error reports probably won't have an error code.

Reply:
ok

Reply:
finish